### PR TITLE
1064 contexts

### LINKED ISSUE #1064: Prezi 3 / Image 2 requires id and @id :(
Document this in the Presentation API.

### PR DESCRIPTION
**Pull request by azaroth42**
Closes #1064 (only editorial, no normative requirements)

### CODE REVIEW

**Comment by zimeon** [comment]
Note conflict with `.gitignore`. Can you summarize the goal of the PR, I read #1064 but am not much wiser

**Comment by azaroth42** [comment]
Sorry, I forgot to change the base branch to prezi3 from master.  Check the changed files now.

**Comment by mikeapp** [comment]
👍

**Comment by azaroth42** [comment]
:+1: to wording simplification
